src/Loadbalancers/FrontendClient.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace UKFast\SDK\Loadbalancers;
+
+use UKFast\SDK\Client;
+use UKFast\SDK\Loadbalancers\Entities\Frontend;
+use UKFast\SDK\SelfResponse;
+
+class FrontendClient extends Client
+{
+    const MAP = [
+        'vips_id' => 'vipsId',
+        'hsts_enabled' => 'hstsEnabled',
+        'hsts_maxage' => 'hstsMaxAge',
+        'redirect_https' => 'redirectHttps',
+        'default_backend_id' => 'defaultBackendId',
+    ];
+
```

Comment:
```suggestion

protected $basePath = 'loadbalancers/';
```

These endpoints are not working as they are missing the basepath. Can this not be extracted and from all the classes and just be set in the base client.